### PR TITLE
Static Page Itsman Integration

### PR DESCRIPTION
**Pull request by iamgarima**

### CODE REVIEW

File: server/handlers/custom-route-handler.js [hunk 1]
```diff
@@ -13,6 +15,38 @@ exports.customRouteHandler = function customRouteHandler(req, res, next, { confi
       if(page.type === 'redirect') { 
         return res.redirect(page["status-code"], page["destination-path"]);
       }
+
+      if(page.type === 'static-page') {
+        if(page.metadata.header || page.metadata.footer) {
```
Comment (gja) [inline server/handlers/custom-route-handler.js]: add `|| page.metadata.layout`.

File: server/handlers/custom-route-handler.js [hunk 2]
```diff
@@ -13,6 +15,38 @@ exports.customRouteHandler = function customRouteHandler(req, res, next, { confi
...
+          const modifiedThemeAttributes = Object.assign({}, config.config["theme-attributes"], { hide_breaking_news: true });
```
Comment (gja) [inline server/handlers/custom-route-handler.js]: don't modify theme attributes.

Comment (gja) [reply server/handlers/custom-route-handler.js]: I don't think you should hide breaking news on static page. Just let it be

File: server/handlers/custom-route-handler.js [hunk 3]
```diff
@@ -13,6 +15,38 @@ exports.customRouteHandler = function customRouteHandler(req, res, next, { confi
...
+            }
+          });
+
+          const seoInstance = (typeof seo == 'function') ? seo(config) : seo;
+          const seoTags = seoInstance && seoInstance.getMetaTags(config, page.type, {}, {url});
+          
+          const cacheKeys = ["static"];
+          addCacheHeadersToResult(res, cacheKeys);
+
+          res.status(page["status-code"]);
```
Comment (gja) [inline server/handlers/custom-route-handler.js]: page["status-code"] || 200.
inline cacheKeys :-p
And you should probably add a cache header with the id of the static page. Probably x/publisher-id/page-id. Check with @AkashHC13 and @codepodu what to use, and also ask them to purge when the static page is updated

File: server/handlers/custom-route-handler.js [hunk 4]
```diff
@@ -13,6 +15,38 @@ exports.customRouteHandler = function customRouteHandler(req, res, next, { confi
...
+
+          return renderLayout(res, {
+            contentTemplate: './custom-static-page',
+            store: store,
+            seoTags: seoTags
+          });
+        }
+        return res.send(page.content);
+      }
```
Comment (amithgeorge) [inline server/handlers/custom-route-handler.js]: I don't see a fallback. If itsman adds a new value for `page.type`, the requests will hang.

Comment (amithgeorge) [reply server/handlers/custom-route-handler.js]: cache keys for rest?

File: server/handlers/custom-route-handler.js (hunk 1 above)
Comment (iamgarima) [reply server/handlers/custom-route-handler.js]: Did we update the data format we receive from the custom-urls api ?

Comment (iamgarima) [reply server/handlers/custom-route-handler.js]: Actually we didn't yet so will ask the platform team to update it but for now i think we have to continue with this.

**Pull request merged**